micro_epsilon_scancontrol_driver/include/micro_epsilon_scancontrol_driver/nodelet.h

```diff
@@ -1,42 +1,43 @@
 #ifndef _SCANCONTROL_NODELET_H_
 #define _SCANCONTROL_NODELET_H_
 
-#include <string>
-#include <boost/thread.hpp>
-
-#include <ros/ros.h>
-#include <pluginlib/class_list_macros.h>
 #include <nodelet/nodelet.h>
+#include <pluginlib/class_list_macros.h>
+#include <ros/ros.h>
+
+#include <boost/thread.hpp>
+#include <string>
 
-#include "micro_epsilon_scancontrol_driver/nodelet.h"
 #include "micro_epsilon_scancontrol_driver/driver.h"
+#include "micro_epsilon_scancontrol_driver/nodelet.h"
 
 namespace scancontrol_driver
 {
-    class DriverNodelet: public nodelet::Nodelet
-    {   
-        public:
-            DriverNodelet():
-                running_(false)
-            {}
-            ~DriverNodelet()
-            {
-                // if (running_)
-                // {
-                //     NODELET_INFO("Shutting down driver thread.");
-                //     running_ = false;
-                //     device_thread_->join();
-                //     NODELET_INFO("Driver thread stopped.");
-                // }
-            }
-
-        private:
-            virtual void onInit();
-
-            volatile bool running_;   
-            boost::shared_ptr<ScanControlDriver> driver_; 
-    };
-
-}
-
-#endif // _SCANCONTROL_NODELET_H_
\ No newline at end of file
+class DriverNodelet : public nodelet::Nodelet
+{
+public:
+  DriverNodelet()
+    : running_(false)
+  {
+  }
+  ~DriverNodelet()
+  {
+    // if (running_)
+    // {
+    //     NODELET_INFO("Shutting down driver thread.");
+    //     running_ = false;
+    //     device_thread_->join();
+    //     NODELET_INFO("Driver thread stopped.");
+    // }
```

Comment:
Not a big deal, just a little note: we're trying to cut down on commented-out code where we can. If there's a reason to keep it, then stick a comment above it explaining why. Otherwise, it should go. :)

Reply:
Will do this in the future tasks where I need to add new functionalities.